recipe/meta.yaml

```diff
@@ -12,21 +12,33 @@ source:
 
 build:
   skip: true  # [py<39 or win]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  script: >-
+    {% if target_platform == 'osx-arm64' %}
+    MESON_ARGS="--cross-file=${RECIPE_DIR}/meson-cross.txt --skip-sanity-check"
```

Comment:
The cross-compiler already sets this to resonable values, it would be better if you don't touch `MESON_ARGS` at all or only extend it with `--skip-sanity-check`

Reply:
It doesn't really work, as you can see from the failed CI runs. I think I tried a lot of possible solutions, but none actually succeeded.

Reply:
Now it works but sadly some Python 3.9 builds fail.